tests/integration_tests/data_steward/validation/main_test.py

```diff
@@ -270,6 +271,16 @@ def test_html_report_five_person(self, mock_check_cron, mock_rdr_date,
         bq_utils.load_table_from_csv(self.project_id, self.rdr_dataset_id,
                                      common.PERSON,
                                      test_util.FIVE_PERSONS_PERSON_CSV)
+
+        # Load measurement_concept_sets
+        required_labs.load_measurement_concept_sets_table(
+            project_id=app_identity.get_application_id(),
+            dataset_id=bq_utils.get_dataset_id())
+        # Load measurement_concept_sets_descendants
+        required_labs.load_measurement_concept_sets_descendants_table(
+            project_id=app_identity.get_application_id(),
+            dataset_id=bq_utils.get_dataset_id())
```

Comment:
nit: These can be replaced with `self.project_id` and `self.dataset_id`

Reply:
Addressed!